README.md

```diff
@@ -63,6 +63,16 @@ fn start
 This will start Fn in single server mode, using an embedded database and message queue. You can find all the
 configuration options [here](docs/operating/options.md). If you are on Windows, check [here](docs/operating/windows.md).
 
+### Quickstart with docker-compose
```

Comment:
Not sure if we should put this in the main readme. From my perspective, it complicates the quickstart, which aims to be as minimalistic as possible. Perhaps it would be better to break it out into docs/operating? Optionally, we could also add a note to the main readme that links to docs/operating, so people know where to find the compose file and helm chart.

Reply:
@denismakogon that's exactly what I said yesterday...